tests/cpu/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+include(ExternalProject)
+
+function(add_ocio_test NAME SOURCES PRIVATE_INCLUDES)
+	set(TEST_BINARY "test_${NAME}_exec")
+	set(TEST_NAME "test_${NAME}")
+	add_executable(${TEST_BINARY} ${SOURCES})
+	target_compile_definitions(${TEST_BINARY}
+		PRIVATE
+			OCIO_UNIT_TEST
+			OpenColorIO_SKIP_IMPORTS
+
+	)
+	target_link_libraries(${TEST_BINARY}
+		PUBLIC
+			public_api
+		PRIVATE
+			tinyxml
+			yamlcpp
+			pystring
+			sampleicc
+			oiio_unittest
+			unittest_data
+	)
+	if(PRIVATE_INCLUDES)
+		target_include_directories(${TEST_BINARY}
+			PRIVATE
+				"${CMAKE_SOURCE_DIR}/src/OpenColorIO"
+		)
+	endif(PRIVATE_INCLUDES)
+	add_test(${TEST_NAME} ${TEST_BINARY})
+endfunction(add_ocio_test)
+
+
+
+# Eventually we will factor out each test into it's own executable
+# but for now, we will maintain the status quo and copy all from the
+# OpenColorIO target
+set(SOURCES
```

Comment:
Same comment as above about hard-coded file names vs. using a glob

Reply:
as above, globbing is evil